verification/verify_premium_ui.py

```
@@ -0,0 +1,48 @@
+            # Click Beacon
+            page.locator('.home-card h3', has_text='Beacon Inbox').click()
+            page.wait_for_timeout(1000)
+
+            # Screenshot Inbox
+            screenshot_path_inbox = "verification/inbox_state.png"
+            page.screenshot(path=screenshot_path_inbox)
+            print(f"Inbox screenshot saved to {screenshot_path_inbox}")
+
+        except Exception as e:
+            print(f"Error: {e}")
+            raise e
```

Ruff (0.14.14)

[warning] Use `raise` without specifying exception name

Remove exception name

(TRY201)